content/fips.md

```diff
@@ -93,13 +94,12 @@ configuration information see [chef-server.rb Optional Settings](/server/config_
 
 ### Prerequisites
 
-- Supported Systems - CentOS, Oracle Linux, or Red Hat Enterprise Linux 6 or later
+- Supported Systems - CentOS, Oracle Linux, or Red Hat Enterprise Linux 6 or later, and Ubuntu
+- Chef Infra Client 16.13 or later for Ubuntu systems
 
 ### Configuration
 
-If you have FIPS compliance enabled at the kernel level then Chef Infra
-Client will default to running in FIPS mode. Otherwise you can add
-`fips true` to the `/etc/chef/client.rb` or `C:\\chef\\client.rb`.
+If you have FIPS compliance enabled at the kernel level then Chef Infra Client will default to running in FIPS mode. Otherwise you can add `fips true` to the `/etc/chef/client.rb` or `C:\\chef\\client.rb`.
```

Comment:
```suggestion
If you have FIPS compliance enabled at the kernel level, then Chef Infra Client will default to running in FIPS mode. Otherwise add `fips true` to the `/etc/chef/client.rb` or `C:\\chef\\client.rb`.
```